lab-jessica/test/test-challenge.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const hanoi = require('../lib/challenge.js');
+const Stack = require('../lib/stack.js');
+const expect = require('chai').expect;
+
+describe('Challenge Module', function() {
+  describe('.setupTower()', function() {
+    it('should put 6 disks on one stack and leave others empty', function() {
+      const stacks = [new Stack(), new Stack(), new Stack()];
+      hanoi.setupTower(6, stacks);
+      expect(stacks[0].size).to.equal(6);
+      expect(stacks[1].size).to.equal(0);
+      expect(stacks[2].size).to.equal(0);
+    });
+
+    it('should have top value of 1 and others null', function() {
```

Comment:
Remember you can also test the `stacks[0].top.prev.prev.prev`, etc to validate that all of the blocks are in the correct order (for a smallish stack anyhow)